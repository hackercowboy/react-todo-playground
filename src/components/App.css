
main {
  max-width: 800px;
  margin: 0 auto;
}

h1 {
  width: 100%;
  text-align: center;
}

.container {
  margin: 20px;
  border: 1px solid #ccc;
}