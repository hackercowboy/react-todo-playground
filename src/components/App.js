import React from 'react';
import Todos from './Todos';

import './App.css';

function App() {
  return (
    <main>
      <h1>Rails Todo Playground</h1>
      <div className="container">
        <Todos />
      </div>
    </main>
  );
}

export default App;
