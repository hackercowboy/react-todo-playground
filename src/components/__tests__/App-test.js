import React from 'react';
import { render, prettyDOM } from '@testing-library/react';

import App from '../App';

describe('<App/>', () => {
  it('should render ', () => {
    render(<App />);
    expect(prettyDOM()).toMatchSnapshot();
  });
});
