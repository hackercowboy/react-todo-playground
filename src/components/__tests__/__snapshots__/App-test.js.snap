// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> should render  1`] = `
"[36m<body>[39m
  [36m<div>[39m
    [36m<main>[39m
      [36m<h1>[39m
        [0mRails Todo Playground[0m
      [36m</h1>[39m
      [36m<div[39m
        [33mclass[39m=[32m\\"container\\"[39m
      [36m>[39m
        [36m<form[39m
          [33maction[39m=[32m\\"#\\"[39m
          [33mclass[39m=[32m\\"todo-form\\"[39m
          [33mdata-testid[39m=[32m\\"todo-form\\"[39m
        [36m>[39m
          [36m<input[39m
            [33mdata-testid[39m=[32m\\"todo-form-input\\"[39m
            [33mid[39m=[32m\\"text\\"[39m
            [33mname[39m=[32m\\"text\\"[39m
            [33mplaceholder[39m=[32m\\"Add new Todo\\"[39m
            [33mtype[39m=[32m\\"text\\"[39m
            [33mvalue[39m=[32m\\"\\"[39m
          [36m/>[39m
        [36m</form>[39m
        [36m<ul[39m
          [33mclass[39m=[32m\\"todo-list\\"[39m
        [36m>[39m
          [36m<li[39m
            [33mclass[39m=[32m\\"todo-list-item\\"[39m
          [36m>[39m
            [36m<span[39m
              [33mclass[39m=[32m\\"todo-list-item-text pending\\"[39m
            [36m>[39m
              [0mLearn React[0m
            [36m</span>[39m
            [36m<span[39m
              [33mclass[39m=[32m\\"todo-list-item-action pending\\"[39m
            [36m>[39m
              [36m<button[39m
                [33mclass[39m=[32m\\"todo-list-item-update-button\\"[39m
                [33mtype[39m=[32m\\"button\\"[39m
              [36m>[39m
                [0mDone[0m
              [36m</button>[39m
            [36m</span>[39m
          [36m</li>[39m
          [36m<li[39m
            [33mclass[39m=[32m\\"todo-list-item\\"[39m
          [36m>[39m
            [36m<span[39m
              [33mclass[39m=[32m\\"todo-list-item-text pending\\"[39m
            [36m>[39m
              [0mBring out the trash[0m
            [36m</span>[39m
            [36m<span[39m
              [33mclass[39m=[32m\\"todo-list-item-action pending\\"[39m
            [36m>[39m
              [36m<button[39m
                [33mclass[39m=[32m\\"todo-list-item-update-button\\"[39m
                [33mtype[39m=[32m\\"button\\"[39m
              [36m>[39m
                [0mDone[0m
              [36m</button>[39m
            [36m</span>[39m
          [36m</li>[39m
          [36m<li[39m
            [33mclass[39m=[32m\\"todo-list-item\\"[39m
          [36m>[39m
            [36m<span[39m
              [33mclass[39m=[32m\\"todo-list-item-text pending\\"[39m
            [36m>[39m
              [0mBuy grocery[0m
            [36m</span>[39m
            [36m<span[39m
              [33mclass[39m=[32m\\"todo-list-item-action pending\\"[39m
            [36m>[39m
              [36m<button[39m
                [33mclass[39m=[32m\\"todo-list-item-update-button\\"[39m
                [33mtype[39m=[32m\\"button\\"[39m
              [36m>[39m
                [0mDone[0m
              [36m</button>[39m
            [36m</span>[39m
          [36m</li>[39m
          [36m<li[39m
            [33mclass[39m=[32m\\"todo-list-item\\"[39m
          [36m>[39m
            [36m<span[39m
              [33mclass[39m=[32m\\"todo-list-item-text done\\"[39m
            [36m>[39m
              [0mMow the lawn[0m
            [36m</span>[39m
            [36m<span[39m
              [33mclass[39m=[32m\\"todo-list-item-action done\\"[39m
            [36m>[39m
              [36m<button[39m
                [33mclass[39m=[32m\\"todo-list-item-delete-button\\"[39m
                [33mtype[39m=[32m\\"button\\"[39m
              [36m>[39m
                [0mDelete[0m
              [36m</button>[39m
            [36m</span>[39m
          [36m</li>[39m
        [36m</ul>[39m
      [36m</div>[39m
    [36m</main>[39m
  [36m</div>[39m
[36m</body>[39m"
`;
